spec.html

```diff
@@ -5871,6 +5871,30 @@ <h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-sametype" type="abstract operation">
+      <h1>
+        SameType (
+          _x_: an ECMAScript language value,
+          _y_: an ECMAScript language value,
+        ): a Boolean
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It determines whether or not the two arguments are the same type.</dd>
+      </dl>
+      <emu-alg>
+        1. If _x_ is *undefined* and _y_ is *undefined*, return *true*.
+        1. If _x_ is *null* and _y_ is *null*, return *true*.
```

Comment:
This could be simplified.

```suggestion
        1. If _x_ is _y_, return *true*.
```

Not sure if that's better or not yet though.

Reply:
Sure, altho that overlaps a bit with all the other cases too, it's still equally correct.

Reply:
I guess our convention is actually to use `SameValue` when all we know is that it's an ECMAScript language value, so in that case, I would just leave it.